apps/users/models.py

```diff
@@ -27,7 +27,11 @@ class User(Base):
     last_visit_at = Column(AwareDateTime, default=func.now(), nullable=False)
     email = Column(VARCHAR(length=100), nullable=False, unique=True)
     is_active = Column(BOOLEAN, default=False)
-    role_id = Column(INTEGER, ForeignKey("role.role_id"), nullable=True)
+    role_id = Column(
+        INTEGER,
+        ForeignKey("role.role_id", ondelete="CASCADE", onupdate="CASCADE"),
```

Comment:
ondelete value isn't correct, i think it should be do nothing or set null...

Reply:
?